docs/configuration.rst

```diff
@@ -699,6 +699,12 @@ PAPERLESS_WEBSERVER_WORKERS=<num>
 
     Defaults to 2.
 
+PAPERLESS_PORT=<port>
+    The port number the webserver will listen on. Change the port if it collides
```

Comment:
We should probably make it clear that this only changes the listening port *inside the countainer* and conflicts on the host should be resolved by modifying `docker-compose.yml`.

Reply:
Sure, go ahead and add it. I'm no expert on docker, so you can likely find the better words for this.

Reply:
I've changed the text. Let me know if you dislike this. Otherwise I'd say LGTM.